lib/kramdown-rfc2629.rb

```diff
@@ -102,10 +102,13 @@ def parse_xref
             when /\A(.*) \((#{SECTIONS_RE})\)\z/
               href = $1
               handle_bares($2, attr, "parens", href)
-            when /\A([\w.]+)<(.*)\z/
+            when /\A(.+)<(.+)\z/
```

Comment:
Why that change?

Reply:
I explained the `[\w.]` to `.` change in the opening comment.

The latter part is so that this pattern doesn't capture `{{foo<}}` only `{{foo<bar}}`.

Reply:
Oh, and I was wrong about the `[\w.]` thing.  Sorry about that, I'll fix it.